packages/ymax-planner/src/axelar/support.ts

```diff
@@ -0,0 +1,55 @@
+import { JsonRpcProvider } from 'ethers';
+import type { EVMContext, EVMChain } from '../subscription-manager';
+
+export const axelarQueryAPI = {
+  mainnet: 'https://api.axelarscan.io/gmp/searchGMP',
+  testnet: 'https://testnet.api.axelarscan.io/gmp/searchGMP',
+};
+
+const { ALCHEMY_KEY } = process.env;
+if (!ALCHEMY_KEY) throw Error(`ALCHEMY_KEY not set`);
```

Comment:
Make sure to document any new environment variables: https://github.com/Agoric/agoric-sdk/tree/master/packages/ymax-planner#configuration

And also to make sure that their purpose is easy to infer. Suggestion: ${NOUN}_API_KEY rather than ${NOUN}_KEY.
```suggestion
const { ALCHEMY_API_KEY } = process.env;
if (!ALCHEMY_API_KEY) throw Error(`ALCHEMY_API_KEY not set`);
```